app/views/root/_base.html.erb

```diff
@@ -44,8 +39,7 @@
 
 <% content_for :content do %>
   <% unless @emergency_banner %>
-    <% show_global_bar = @banner_notification.blank? && !local_assigns[:hide_banner_notification] && !@emergency_banner %>
-    <%= render partial: "promo_banner" if show_global_bar %>
+    <%= render partial: "promo_banner" if !@emergency_banner %>
```

Comment:
You don't need `if !@emergency_banner` here, it's handled by the `unless` in the previous line.